### ISSUE TITLE
This NPC is not belong to npc/re/instances/WolfchevLaboratory.txt

### ISSUE THREAD

**Issue by cydh** [rathena/rathena#774]
I just tried to finding rA script while playing idRO. Found the reward NPC technically is not instance dependent.
```
// Weird Old Man (Rewards)
//============================================================
lighthalzen,342,291,4	script	Weird old man#Bio4Reward	4_M_EINOLD,{
}
```
Why, because the required items can be dropped from lhz_dun03 and lhz_dun04, not only from `1@lhz` instance.

and also with this enchant NPC
```
// Sorcerer (Rewards)
//============================================================
lhz_cube,233,24,4	script	Sorcerer#Bio4Reward	4_M_UMDANCEKID,{
}
```

So, if user want disable instance but just enable the quest, when instance file is commented, the quest is still.

**Comment by Lemongrass3110** [comment]
Edited your text to match Github's flavored markdown.

**Comment by Lemongrass3110** [comment]
Fixed in 88e5220

**Comment by Lemongrass3110** [comment]
Merged this and added the loading in 1f26ad5.
